src/morale_types.h

```diff
@@ -56,6 +56,7 @@ enum morale_type : int {
     MORALE_PERM_MASOCHIST,
     MORALE_PERM_HOARDER,
     MORALE_PERM_FANCY,
+    MORALE_PERM_FILTHY,
```

Comment:
This will break current saves. Place the new morale type at the very bottom, right before `NUM_MORALE_TYPES`.